packages/scandipwa/src/component/CheckoutBilling/CheckoutBilling.component.js

```diff
@@ -133,20 +135,40 @@ export class CheckoutBilling extends PureComponent {
         );
     }
 
+    renderOrderTotalExlTax() {
+        const {
+            cartTotalSubPrice,
+            totals: { quote_currency_code }
+        } = this.props;
+
+        if (!cartTotalSubPrice) {
+            return null;
+        }
+
+        const orderTotalExlTax = formatPrice(cartTotalSubPrice, quote_currency_code);
+
+        return (
+            <span>
+                { `${ __('Excl. tax:') } ${ orderTotalExlTax }` }
+            </span>
+        );
+    }
+
     renderOrderTotal() {
         const { totals: { grand_total, quote_currency_code } } = this.props;
 
         const orderTotal = formatPrice(grand_total, quote_currency_code);
 
         return (
-            <div block="Checkout" elem="OrderTotal">
-                <span>
+            <dl block="Checkout" elem="OrderTotal">
+                <dt>
                     { __('Order total:') }
-                </span>
-                <span>
+                </dt>
+                <dd>
                     { orderTotal }
-                </span>
-            </div>
+                    { this.renderOrderTotalExlTax() }
+                </dd>
+            </dl>
```

Comment:
Since you have a method to render total excl tax, I propose having a separate one for the incl tax also. So that the partial API of this class offers:
- `renderOrderTotal` - calls the two methods above and marks up the output
- `renderOrderTotalExclTax`
- `renderOrderTotalInclTax`